lib/ttt_app.rb

```diff
@@ -0,0 +1,11 @@
+module TttApp
+  class Error < StandardError; end
+  # Your code goes here...
+  print('Welcom To TTT')
+end
+
+class TicTacToe
+  def self.printWelcome
+      'Welcome to Tic Tac Toe Game'
```

Comment:
To print out a string into the console, you can use `puts` or `print`.

Reply:
Noted